src/components/ActionsPanel/constants.ts

```diff
@@ -279,11 +338,107 @@ export const actionTemplates: ActionTemplate[] = [
     body: [translate('TWITTER_ACTION_BODY')],
     priority: 0,
     button: {
-      content: translateRaw('TWITTER_ACTION_BUTTON'),
-      shouldComplete: true,
-      to: socialMediaLinks.find((i) => i.text === 'twitter')!.link,
-      external: true
+      component: ActionButton,
+      props: {
+        content: translateRaw('TWITTER_ACTION_BUTTON'),
+        shouldComplete: true,
+        to: socialMediaLinks.find((i) => i.text === 'twitter')!.link,
+        external: true
+      }
     },
     category: ACTION_CATEGORIES.OTHER
+  },
+  {
+    name: ACTION_NAME.MYC_WINTER_START,
+    icon: 'myc-winter',
+    heading: translateRaw('MYC_WINTER_START_ACTION_HEADING'),
+    body: [
+      translate('MYC_WINTER_START_ACTION_BODY_1'),
+      translate('MYC_WINTER_START_ACTION_BODY_2')
+    ],
+    priority: 0,
+    time: {
+      start: new Date('2020-12-01'),
+      end: new Date('2020-12-12')
+    },
+    button: {
+      component: ActionButton,
+      props: {
+        content: translateRaw('MYC_WINTER_START_ACTION_BUTTON'),
+        shouldComplete: true,
+        to: MYC_WINTER_LINK,
+        external: true
+      }
+    },
+    category: ACTION_CATEGORIES.SELF_LOVE
+  },
+  {
+    name: ACTION_NAME.MYC_WINTER_MID,
+    icon: 'myc-winter',
+    heading: translateRaw('MYC_WINTER_MID_ACTION_HEADING'),
+    subHeading: GeneralSubHead,
+    props: {
+      content: translateRaw('MYC_WINTER_MID_ACTION_SUBHEAD')
+    },
+    body: [translate('MYC_WINTER_MID_ACTION_BODY_1'), translate('MYC_WINTER_MID_ACTION_BODY_2')],
+    priority: 0,
+    time: {
+      start: new Date('2020-12-12'),
+      end: new Date('2020-12-20')
```

Comment:
Are the ends and starts supposed to overlap?

Reply:
Yeah the next one should start when the previous one ends

Reply:
But will a new one not start on the same day as another ends, effectively giving two actions?